pkg/probe/http/http.go

```diff
@@ -107,7 +111,7 @@ func DoHTTPProbe(url *url.URL, headers http.Header, client GetHTTPInterface) (pr
 		return probe.Failure, err.Error(), nil
 	}
 	defer res.Body.Close()
-	b, err := ioutil.ReadAll(res.Body)
+	b, err := sysutils.ReadAllWithLimit(res.Body, maxReadLength)
```

Comment:
Similar usecase to the lifecycle handler. I suggest making the same changes here.

Reply:
Done